src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -13,7 +13,7 @@ OverlayWindow* instance = nullptr;
 
 namespace
 {
-    LRESULT CALLBACK hookProc(int nCode, WPARAM wParam, LPARAM lParam)
+    LRESULT CALLBACK lowLevelKeyboardProc(int nCode, WPARAM wParam, LPARAM lParam)

```

Comment:
We should use upper case for this function, moving forward to new standards for naming functions and breaking the old style.

Reply:
Fixed